src/Raven.Server/Documents/Handlers/BatchRequestParser.cs

```diff
@@ -805,33 +808,73 @@ private static unsafe CommandPropertyName GetPropertyType(JsonParserState state)
                         *(int*)(state.StringBuffer + sizeof(long)) == 1231974249 &&
                         state.StringBuffer[12] == (byte)'d')
                         return CommandPropertyName.DestinationId;
-
                     return CommandPropertyName.NoSuchProperty;
+                
+                case 14:
+                    if (*(int*)state.StringBuffer == 1668571472 &&
+                        *(long*)(state.StringBuffer + sizeof(int)) == 7598543892411468136 &&
+                        *(short*)(state.StringBuffer + sizeof(int) + sizeof(long)) == 26478)
+                        return CommandPropertyName.PatchIfMissing;
+                    if (*(int*)state.StringBuffer == 1970562386 &&
+                        *(long*)(state.StringBuffer + sizeof(int)) == 7308626840221150834 &&
+                        *(short*)(state.StringBuffer + sizeof(int) + sizeof(long)) == 29806)
+                        return CommandPropertyName.ReturnDocument;
+                    if (*(int*)state.StringBuffer == 1635021889 &&
+                        *(long*)(state.StringBuffer + sizeof(int)) == 8742740794129868899 &&
+                        *(short*)(state.StringBuffer + sizeof(int) + sizeof(long)) == 25968)
+                        return CommandPropertyName.AttachmentType;
+                    return CommandPropertyName.NoSuchProperty;
+                
                 case 15:
                     if (*(long*)state.StringBuffer == 8386105380344915268 &&
                         *(int*)(state.StringBuffer + sizeof(long)) == 1315860329 &&
                         *(short*)(state.StringBuffer + sizeof(long) + sizeof(int)) == 28001 &&
                         state.StringBuffer[14] == (byte)'e')
                         return CommandPropertyName.DestinationName;
-
                     return CommandPropertyName.NoSuchProperty;
+                
+                case 29:
+                    if (*(long*)state.StringBuffer == 8531315664536891206 &&
+                        *(long*)(state.StringBuffer + sizeof(long)) == 7309979286770381673 &&
+                        *(long*)(state.StringBuffer + sizeof(long) + sizeof(long)) == 8247308551402910817 &&
+                        *(int*)(state.StringBuffer + sizeof(long) + sizeof(long) + sizeof(long)) == 1734702177 &&
+                        state.StringBuffer[28] == (byte)'y')
+                        return CommandPropertyName.ForceRevisionCreationStrategy;
+                    return CommandPropertyName.NoSuchProperty;
+                
                 default:
                     return CommandPropertyName.NoSuchProperty;
             }
         }
 
+        private static unsafe ForceRevisionStrategy GetEnumValue(JsonParserState state, JsonOperationContext ctx)
+        {
+            switch (state.StringSize)
+            {
+                case 6:
+                    if (*(int*)state.StringBuffer == 1868981570 &&
+                        state.StringBuffer[5] == (byte)'e')
```

Comment:
Missing check for `StringBuffer[4]`, need to compare it to `short` value, I think.

Reply:
Fixed. Used int & short.